pkgs/games/among-sus/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, port ? "1234" }:
+
+stdenv.mkDerivation {
+  pname = "among-sus";
+  version = "2020-10-22";
+
+  src = fetchgit {
+    url = "https://git.sr.ht/~martijnbraam/among-sus";
+    rev = "fb2bd86344cf07b65383d12fe6998b4f895f8fab";
+    sha256 = "0pli5554zg9x5zv6hi2z85dzcvw1xxcl0saj3i1a8f8z6135hxfd";
+  };
+
+  patchPhase = ''
+    sed -i 's/port = 1234/port = ${port}/g' main.c
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv among-sus $out/bin
+  '';
+
+  meta = {
+    homepage = "https://git.sr.ht/~martijnbraam/among-sus";
+    description = "Among us, but it's a text adventure";
+    license = stdenv.lib.licenses.agpl3Plus;
+    maintainers = with maintainers; [ eyjhb ];
+  };
```

Fixes the ofborg failure:
```suggestion
  meta = with stdenv.lib; {
    homepage = "https://git.sr.ht/~martijnbraam/among-sus";
    description = "Among us, but it's a text adventure";
    license = licenses.agpl3Plus;
    maintainers = [ maintainers.eyjhb ];
  };
```